src/pieces/assets/assets_command.py

```diff
@@ -5,7 +5,17 @@
 from .assets_api import AssetsCommandsApi
 from pieces.gui import show_error,print_model_details,space_below,double_line,deprecated
 from pieces.settings import Settings
-
+import subprocess
+from prompt_toolkit import PromptSession
+from prompt_toolkit.document import Document
+from prompt_toolkit.key_binding import KeyBindings
+from pygments import highlight
+from pygments.lexers import get_lexer_by_name, guess_lexer
+from pygments.formatters import TerminalFormatter
+import json
+import tempfile
+
+CONFIG_FILE = os.path.expanduser("~/.pieces_config.json")
```

Comment:
we already have a directory where we save these configurations it was Settings.data_path I believe

Reply:
could you provide me with the code where this is being referenced or called or defined ? I tried looking to up in the repo but couldn't find the data_path anywhere . I did encounter the pieces_data_dir , BASE_DIR , extensions_dir and open_snippet_dir though . so any way of calling / referencing the data_path would be helpful .

Also , probably unrelated but on settings.py line 56 mentions a configuration.py file , which I could not see anywhere , so I was wondering if this was something I was missing on my part or if it was something that had existed in the past ?